pkg/state/cluster.go

```diff
@@ -74,6 +77,27 @@ const (
 	Cluster level checks
 */
 
+const (
+	x90Days = time.Minute * 24 * 90
```

Comment:
You're missing `* 60` for the number of minutes in one hour. You can do something like this instead:

```suggestion
	x90Days = time.Hour * 24 * 90
```

Reply:
ups :D good you’re paying attention :) 👍
done